nupic/torch/modules/sparse_weights.py

```diff
@@ -64,21 +65,30 @@ class SparseWeightsBase(nn.Module, metaclass=abc.ABCMeta):
 
     :param module:
       The module to sparsify the weights
-    :param weight_sparsity:
-      Pct of weights that are allowed to be non-zero in the layer.
+    :param sparsity:
+      Pct of weights that are zero in the layer.
     """
 
-    def __init__(self, module, weight_sparsity):
+    def __init__(self, module, weight_sparsity=None, sparsity=None):
         super(SparseWeightsBase, self).__init__()
-        assert 0 < weight_sparsity < 1
+        assert weight_sparsity is not None or sparsity is not None
+        if weight_sparsity is not None and sparsity is None:
+            assert 0 < weight_sparsity < 1
+            sparsity = 1.0 - weight_sparsity
+            warnings.warn(
+                "Parameter `weight_sparsity` is deprecated. Use `sparsity` instead.",
+                DeprecationWarning,
+            )
+
+        assert 0 < sparsity < 1
 
         self.module = module
-        self.weight_sparsity = weight_sparsity
+        self.sparsity = sparsity
```

Comment:
For backward compatibility purpose, just in case someone access `weight_sparsity` directly from the instance.
could you also add `weight_sparsity` as a property to this class returning `1.0 - self. sparsity` and giving the same warning?